src/main/java/report_screens/FileReportHistory.java

```diff
@@ -106,5 +106,13 @@ public boolean existsReportByReporterAndReview(String reporter_username, String
         String corresponding_reporter = check_reports.get(review_id);
         return Objects.equals(corresponding_reporter, reporter_username);
     }
+
+    public Map<String, ReportDsRequestModel> getSave_reports() {
+        return save_reports;
+    }
+
+    public Map<String, String> getCheck_reports() {
+        return check_reports;
+    }
```

Comment:
Consider renaming check_reports to reviewOwner or something like that so the variable says what it contains rather what it is used for.

Reply:
sure